src/components/post/Post.stories.tsx

```diff
@@ -0,0 +1,11 @@
+import { Meta, Story } from "@storybook/react";
+import Post from "./Post";
+
+export default {
+  title: "Post",
+  component: Post,
+} as Meta;
+
+const Template: Story = (args) => <Post {...args} />;
+
+export const Default = Template.bind({});
```

Comment:
should add more scenarios, such as:

"without image"
"with really long title"
"with very short title"
"with empty description"